src/js/controller/controller.js

```diff
@@ -601,7 +618,7 @@ Object.assign(Controller.prototype, {
         }
 
         function _isIdle() {
-            const state = _model.get('state');
+            const state = _getState();
```

Comment:
`_getState` includes ad state. Are you sure you want to change `_model.get('state')` to `_getState()`?